src/test/java/org/cactoos/package-info.java

```diff
@@ -25,7 +25,7 @@
 /**
  * Tests.
  *
- * @todo #1420:30min Continue replacing usage of MatcherAssert.assertThat with
+ * @todo #1425:30min Continue replacing usage of MatcherAssert.assertThat with
```

Comment:
@baudoliver7 
Is it possible to update this list now?
https://github.com/yegor256/cactoos/blob/master/pom.xml#L266

Reply:
@andreoss Not for the moment because `iterable` package is not finished yet. I made a part of this package to keep this PR small.